src/app/game/game.component.ts

```diff
@@ -7,9 +8,22 @@ import {Component, OnInit} from '@angular/core';
 })
 export class GameComponent implements OnInit {
 
-  constructor() {
+  firstBotName: string;
+  secondBotName: string;
+  gameId: number;
+
+  constructor(private readonly route: ActivatedRoute,
+              private router: Router) {
+    this.router.routeReuseStrategy.shouldReuseRoute = () => false;
   }
 
   ngOnInit(): void {
+
+    this.route.params.subscribe(params => this.gameId = +params.id);
```

Comment:
What kind of operator is this `= +` ?

Reply:
Cast value to number and assign to variable in one 😄